packages/admin/cms-admin/src/pages/pagesPage/PagesPageActionToolbarBatchUpdateHelper.ts

```diff
@@ -28,3 +28,22 @@ export const pageTreeBatchUpdateVisibility = async (
     );
     return;
 };
+
+export const pageTreeBatchResetVisibility = async (
+    client: ApolloClient<unknown>,
+    pageTreeNodesWithPreviousVisibility: Pick<GQLPageTreePageFragment, "id" | "visibility">[],
```

Comment:
@johnnyomair I renamed this from `pageTreeNodes` to `pageTreeNodesWithPreviousVisibility`. Do you think it's clearer that way?